schema.ts

```diff
@@ -1565,6 +1565,7 @@ export function normalizeSchema<T extends Schema>(schema: T): T {
     const normalized: ObjectSchemaProperties = {};
     const {
       id,
+      fixedId,
```

Comment:
Can do later, but given how dangerous this method is proving to be, we should use the new exhaustiveness-checking technique and destructure all of the properties of this schema, so TS will catch if a new property is added but not handled here.

It would amount to adding this here:

```typescript
      identity,
      description,
      attribution,
      fixedId,
      type,
      codaType,
      properties,
      includeUnknownProperties,
      autocomplete,
      mutable,
      // eslint-disable-next-line @typescript-eslint/naming-convention
      __packId,
      ...rest
    } = schema as GenericObjectSchema;
    ensureNever<keyof typeof rest>();
```

and then using those destructured fields below instead of referencing them as `schema.foo`.

Reply:
Added! I went ahead and alphabetized the property lists too, for faster scanning.